build.gradle

```diff
@@ -4619,18 +4557,16 @@ task javadoc(type: Javadoc, dependsOn: createMSPfile) {
         // Add an entry for each incubator module here, leaving the incubator
         // placeholder lines as an example.
         // BEGIN: incubator placeholder
-        //project(":incubator.mymod"),
-        // END: incubator placeholder
         project(":incubator.input"),
         project(":incubator.richtext"),
+        // END: incubator placeholder
```

Comment:
This part was correct before. Please revert. It should look like this when you are done:

```
        // Add an entry for each incubator module here, leaving the incubator
        // placeholder lines as an example.
        // BEGIN: incubator placeholder
        //project(":incubator.mymod"),
        // END: incubator placeholder
        project(":incubator.input"),
        project(":incubator.richtext"),
```